backend/development.ini.sample

```diff
@@ -38,6 +48,10 @@ basic_setup.session_secret = change_this_value_please!
 # CORE
 ####
 
+# hapic debug mode: return python traceback of error
+; debug = False
```

Comment:
This parameter only configure hapic debug mode ? If yes, this is a shadow config name no ?

Reply:
"shadow" ?

Reply:
> "shadow" ?

https://en.wikipedia.org/wiki/Variable_shadowing#Python

I mean: `debug` name look like a config to enable debug on whole Tracim. But, like indicated by comment, it enable only debug for hapic.

Reply:
En fait y'a pas de variable shadowing car on à pas d'autre debug dans tracim.

Reply:
le reste du debug c'est la configuration des logger c'est autre chose.